index.js

```diff
@@ -1708,6 +1711,28 @@ function humanReadableArgName(arg) {
     : '[' + nameOutput + ']';
 }
 
+/**
+ * Parse the short and long flag out of something like '-m,--mixed <value>'
+ *
+ * @api private
+ */
+
+function _parseOptionFlags(flags) {
+  let shortFlag;
+  let longFlag;
+  // Use original very loose parsing to maintain backwards compatibility for now,
+  // which allowed for example unintended `-sw, --short-word` [sic].
+  const flagParts = flags.split(/[ |,]+/);
```

Comment:
this pattern seems buggy if the comment is still accurate, did you intend to say `/( |,)+/` there? or just `/[ ,]+/` to match space or comma, or really wanted to add a pipe `|` to the mix?

Reply:
The pipe is really in the mix. From the README:

> Each option can have a short flag (single character) and a long name, separated by a comma or space or vertical bar ('|').

The comment is about `-sw` which is neither a short flag nor a long name.